docs/Getting Started/Connecting to an Infotainment System/index.md

```diff
@@ -99,24 +109,52 @@ If you are using Sdl Core and Generic HMI, you will have to add a policy table e
+For more information about policy tables please visit the [Policy Table](https://smartdevicelink.com/en/guides/sdl-server/api-reference-documentation/policy-table/overview) guide.
+
+### Manticore
+If you are using Manticore, the app connection information can be easily added in the settings tab of the Manticore web page. Please note that Manticore needs to access your machine's IP address in order to be able to start a websocket connection with your app. If you are hosting the app on your local machine, you may need to do extra setup to make your machine publicly accessible.
+
+![Manticore - Cloud and Embedded App Settings](assets/Manticore_cloud_embedded_app_settings.png)
+
+## Running the SDL App
+Once you have a configured instance of Core running, you should see your SDL app name and icon appear on HMI. However, nothing will happen when you tap on your SDL app icon until you build and run your SDL app.
+
+![Generic - SDL Apps Tab](assets/Generic_apps_screen.png)
+
+Once your SDL app is running, either locally in an IDE or on a server, you will be able to launch the SDL app by clicking on the app icon in the HMI.
+
+![Generic - SDL App Main Screen](assets/Generic_non_media.png)
+
+This is the main screen of your SDL app. If you get to this point, your SDL app is working.
+!@
+
+## How Do I Find Out What Features My Infotainment System Supports?
+New features are always being added to SDL Core, however, you or your users may be connecting to older head units that do not support the newest features. If your SDL app attempts to implement an unsupported feature, your request will simply be ignored by SDL Core.
 
-!!! NOTE
-Don't forget to replace `ws://<ip address>:<port>` with your own IP address and app port. Also `<app name>` should be replaced with the actual app name.
-!!!
+### How to Handle Unsupported Features
+When you are implementing a feature, you should always assume that some of the head units that your users are connecting to will not support the feature or that your user has disabled permission to use this feature on their head unit. The best way to deal with unsupported features is to handle unsuccessful responses to the requests you sent to SDL Core. 
 
-Following this, you should see an application appears on HMI as in the following screenshot:
+### Checking the RPC Spec Version
+When you connect successfully to a head unit, you can get the most current [RPC Spec](https://github.com/smartdevicelink/rpc_spec/blob/master/MOBILE_API.xml) version that both the head unit and your mobile SDL SDK support. If the feature you want to support was added in version less than or equal to the version returned by the head unit, then your head unit supports the feature. However, please remember that you may still need permission from the OEM to use the feature.  
 
-![HMI Apps](assets/hmi1.png)
+@![iOS]
+##### Objective-C
+```objc
+NSString *rpcSpecVersion = self.sdlManager.registerResponse.sdlMsgVersion.description;
```

Comment:
Don't have them pull a string like this. Have them pull the version object. This is also the wrong place to pull the version, it should be pulled from `SDLGlobals`.

Reply:
`SDLGlobals` header file is private...

Reply:
:( I've thought for the longest time that it was public.